planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -163,6 +165,11 @@ void planning_scene_monitor::PlanningSceneMonitor::initialize(const planning_sce
   moveit::tools::Profiler::ScopedStart prof_start;
   moveit::tools::Profiler::ScopedBlock prof_block("PlanningSceneMonitor::initialize");
 
+  // start our own spinner listening on our own callback_queue to become independent of any global callback queue
+  root_nh_.setCallbackQueue(&callback_queue_);
+  spinner_.reset(new ros::AsyncSpinner(1 /* threads */, &callback_queue_));
+  spinner_->start();
```

Comment:
I think this is probably a nice fix. Maybe we should allow the user to specify how many threads he wants to use.
The user does not really see more than this since it is encapsulated in the move_group node, so it is probably more a matter of how many more threads does he allow on his computer.